_includes/header.html

```diff
@@ -389,20 +369,117 @@
 a:hover {
 	text-decoration: underline;
 }
-#feet{
-	display:flex;
-	wrap: nowrap;
-
 
+  .top {
+  background-image: url(images/fossasia2016.jpg);
```

Comment:
Missing indentation here and same in many lines below. Also, I can see 2 commits, make sure only 1 commit remains after squashing.